conda/cli/main_clean.py

```diff
@@ -65,6 +72,66 @@ def configure_parser(sub_parsers):
     p.set_defaults(func=execute)
 
 
+# work-around for python bug on Windows prior to python 3.2
```

Comment:
@kalefranz Actually, python 3.2 works just fine. This is needed for all earlier python versions, e.g. python 2.7, etc.